packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/sagas.ts

```diff
@@ -418,7 +418,12 @@ export default ({ api, coreSagas, networks }) => {
             break
           }
           sddVerified = yield call(api.fetchSDDVerified)
-          if (sddVerified?.taskComplete) break
+          if (sddVerified?.taskComplete) {
+            yield put(
+              actions.components.simpleBuy.fetchSDDVerifiedSuccess(sddVerified)
+            )
```

Comment:
we got response that user is verified, flag this in SimpleBuy flow (maybe we should move this to more common place)